src/main/java/com/googlecode/lanterna/gui2/Window.java

```diff
@@ -390,7 +398,11 @@ public String toString() {
      * @return The global coordinates expressed as local coordinates
      */
     @Override
+    @Deprecated
```

Comment:
Add a @deprecated Javadoc note for this and explain what to use instead

Reply:
I've updated with some javadocs.